test/Shared/UserManagerTestBase.cs

```diff
@@ -1165,19 +1174,19 @@ public Task<IdentityResult> ValidateAsync(UserManager<TUser> manager, TUser user
             var context = CreateTestContext();
             var manager = CreateManager(context);
             var roleManager = CreateRoleManager(context);
-            var role = CreateRole("addUserTest");
+            var roleName = "AddUserTest" + Guid.NewGuid().ToString();
```

Comment:
Yeah these role + guid name lines are all over, move this into the helper just like CreateTestUser